imports/plugins/included/ui-search/lib/components/searchModal.js

```diff
@@ -1,6 +1,7 @@
 import React, { Component } from "react";
```

Comment:
I believe that you shouldn't have this file in your PR changes as you haven't changed anything in it. Please revert to what is on staging, i.e. copy and paste the staging contents into this file